src/pyj/read_book/cfi.pyj

```diff
@@ -57,21 +57,6 @@ def window_scroll_pos(w):  # {{{
     return w.pageXOffset, w.pageYOffset
 # }}}
 
-def viewport_to_document(x, y, doc):  # {{{
-    doc = doc or window.document
-    while doc is not window.document:
-        # we are in a frame
-        frame = doc.defaultView.frameElement
-        rect = frame.getBoundingClientRect()
-        x += rect.left
-        y += rect.top
-        doc = frame.ownerDocument
-    wx, wy = window_scroll_pos(doc.defaultView)
-    x += wx
-    y += wy
-    return x, y
-# }}}
-
```

Comment:
ScrollViewport now has contextual information about the viewport, namely whether things are in RTL mode, so it needs to wrap this to make sure the returned coordinates are in the correct space.